main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsNetworkSchema.java

```diff
@@ -0,0 +1,15 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+@GtfsTable("networks.txt")
+@Required
+public interface GtfsNetworkSchema extends GtfsEntity {
+  @FieldType(FieldTypeEnum.ID)
+  @PrimaryKey
+  @Required
```

Comment:
Should be optional field, not required, and add conditionally forbidden logic in a separate PR. Forbidden if network_id exists in [routes.txt](https://gtfs.org/schedule/reference/#routestxt). cc @jcpitre

Reply:
As discussed in the standup, @jcpitre can take care of this so I can finish the load test for Mobility Database API!

Reply:
Delegated to another PR

Reply:
Before merging let's remove the `Required` annotations.

Reply:
It's already removed. Ready to be merged.